processing/data_collection/gazette/spiders/rj_campos_goytacazes.py

```diff
@@ -0,0 +1,45 @@
+import re
+import dateparser
+
+from datetime import datetime
+from scrapy import Request
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+class RjCampoGoytacazesSpider(BaseGazetteSpider):
+    GAZETTE_ELEMENT_CSS = 'ul.ul-licitacoes li'
+    MUNICIPALITY_ID = '3301009'
+
+    allowed_domains = ['www.campos.rj.gov.br']
+    name = 'rj_campos_goytacazes'
+    start_urls = ['https://www.campos.rj.gov.br/diario-oficial.php?PGpagina=1&PGporPagina=15']
+
+    def parse(self, response):
+        """
+        @url https://www.campos.rj.gov.br/diario-oficial.php?PGpagina=1&PGporPagina=15
+        @returns requests 1
+        @scrapes date file_urls is_extra_edition municipality_id power scraped_at
+        """
+
+        for element in response.css(self.GAZETTE_ELEMENT_CSS):
+            url = element.css('a::attr(href)').extract_first()
+            gazette_text = element.css("h4::text").extract_first()
+            extra_edition = gazette_text.startswith('Suplemento')
+
+            date_re = re.search(r'(\d{2} de (.*) de \d{4})', gazette_text)
+            if date_re:
```

Comment:
Did you see some gazette without date or in a different format?

Reply:
The first list item of the list has no date.